README.md

```diff
@@ -30,34 +30,21 @@ When working with [OpenShift Online](https://www.openshift.com/products/online/)
 
 ## New and Noteworthy
 
-## All Required CLI Tools Included
-
-This release includes binaries for all required CLI tools:
-* OKD CLI Client (`oc`)
-* OpenShift Do (`odo`)
-Once extension is installed it is ready to use. There is no additional configuration steps to download CLI tools binaries.
-
-### Debug Support for Local Node.js and Java Components
-
-This release provides new 'OpenShift: Debug' command to simplify the way to start debugging for OpenShift Components pushed to a cluster. It is an experimental feature, because it is using experimental OpenShift Do `debug` command under the hood and supports only local Java and Node.js components. The command is available from command palette and context menu for Component nodes in OpenShift Application Explorer view. 
-
-#### Debug Node.js Component
-
-Default Visual Studio Code installation includes JavaScript/TypeScript Language Support and Debugger Extensions required to debug a Node.js Component. That means new `OpenShift: Debug` command can be used without installing any additional extensions.
-
-![ screencast ](https://raw.githubusercontent.com/redhat-developer/vscode-openshift-tools/master/images/debug-node.gif)
-
-#### Debug Java Component
-
-To debug a Java Component, Java Language Support and Java Debugger Extensions are required. OpenShift Connector extension will prompt the user to install missing extension(s) before it starts Debugger for a Java Component.
-
-![ screencast ](https://raw.githubusercontent.com/redhat-developer/vscode-openshift-tools/master/images/debug-java.gif)
+* Latest OpenShift Do CLI tool v1.2.1 included 
```

Comment:
```suggestion
* Supports OpenShift Do CLI tool [v1.2.1]((https://github.com/openshift/odo/releases/tag/v1.2.1) )
```

Reply:
@dgolovin The reason to not have `latest` word is because the odo version will not be latest always to 1.2.1 till the time this version of extension is used. So supports is a better keyword to demonstrate here.